README.md

```diff
@@ -1 +1,109 @@
-# catalog-of-my-things
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Live Video](#live-video)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+<!-- PROJECT DESCRIPTION -->
+
+# Catalog <a name="about-project"></a>
+
+**Catalog** is a project where you can add, and consult for different types of things like: books, music albums, movies, and games. Following the next UML class diagram:
+
+<p align="left">
+  <img src="./catalog_of_my_things.png" alt="C=UML class diagram for catalog of things" height="350" width="500" />
+</p>
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Developed using Ruby.
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+- **Database that contains information about a catalogue**
+
+### Live Video <a name="live-video"></a>
+- Click [here](https://drive.google.com/file/d/12NXPuV4G5YmCBKunUCU91HDlEXYAsty6/view?usp=sharing) to see live video.
+
+## 💻 Getting Started <a name="getting-started"></a>
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- You need to have git installed in your machine. Again you need to have `npm`, `node` installed in your computer.
+
+### Setup
+```
+- To clone the repository run this command - "git clone https://github.com/Haywayaheadshot/catalog-of-my-things.git"
+- cd catalog
+```
```

Comment:
- [ ] After setting up, kindly add information that will make it easy to run the app. For example, you should add `ruby main.rb` command to start the application in the console. 👍